net/raft/http.go

```diff
@@ -161,6 +168,17 @@ func sendmsg(addr string, data []byte, client *http.Client) {
 	defer resp.Body.Close()
 }
 
+// Sends eviction notice to a node to kill the process
+func evictionNotice(addr string, client *http.Client) {
+	url := "https://" + addr + "/raft/kill"
+	resp, err := client.Post(url, contentType, nil)
+	if err != nil {
+		log.Printkv(context.Background(), "warning", err)
+		return
+	}
+	defer resp.Body.Close()
+}
```

Comment:
Just updated to have nodes recognize evictions when they get a response that they're no longer in the cluster — thanks!